tests/workflows/parallel_node_cancellation/workflow.py

```diff
@@ -0,0 +1,31 @@
+import time
```

Comment:
we need empty `__init__.py` files in this directory and the `tests/workflows/parallel_node_cancellation/tests` directory